src/features/jserrors/aggregate/index.js

```diff
@@ -199,7 +201,8 @@ export class Aggregate extends FeatureBase {
 
     // stn and spa aggregators listen to this event - stn sends the error in its payload,
     // and spa annotates the error with interaction info
-    handle('errorAgg', [type, bucketHash, params, newMetrics], undefined, undefined, this.ee)
+    handle('errorAgg', [type, bucketHash, params, newMetrics], undefined, FEATURE_NAMES.sessionTrace, this.ee)
+    handle('errorAgg', [type, bucketHash, params, newMetrics], undefined, FEATURE_NAMES.spa, this.ee)
```

Comment:
note that these buffers are storing to other buckets.  This is useful to know because these buckets belong to 'sessionTrace' and 'spa' but are being filled by data in jserrors